### PR TITLE
New design!

### PR DESCRIPTION
**Pull request by MrTamagotchi**
So I spent the most of my day designing and styling what we have so far. Except for new colors, I changed the layout and added some new data. I have looked at the list of hopeful features and decided to go for a very modular design, I'm thinking that the user section could be decoupled from the rest and used when listing users only.

Finding a suiting mentor/mentoree(?)(manatee??) does not only require the right skillset, but also somewhat matching personalities. I decided to make the profile image a bit bigger (since a picture tells more than a thousand words) and also include the geolocation and caption from the fff-usermap.

The thought is to actually have the location be humanly readable in the future, and also link to the pin in the map (will probably need to modify the map for that?). It only links to the whole map as of now.

Other changes include some reorganizing in the styles, in anticipation for app growth.

There's a small FIX ME in UserTile.less where I resort to global styling of the `.skill`, because I've never worked with css-modules before and I ran out of googlable terms. How do you modify children based on parent classes in a locally scoped way?

Anyways, hope you like this! 😎

### CODE REVIEW

**Comment by ChemicalRocketeer** [comment]
This is way cool, 💯 Love the new look.

Only thing is the test fixtures need to be updated to include the new geolocation stuff, so that the tests pass. I'm going to work on getting some CI stuff set up and integrated with github so that it's more obvious when a PR breaks tests.

File: src/components/global.less [hunk 1]
```diff
@@ -4,31 +4,18 @@
...
+:global body {
+  background: @color-background;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
 }
 
-@profile-pic-tiny-width: 28px;
+@profile-pic-tiny-width: 80px;
 
 :global .profile-tiny {
```
Comment (ChemicalRocketeer) [inline src/components/global.less]: This maybe shouldn't be called profile-tiny anymore, lol

File: src/components/UserTile.jsx [hunk 2]
```diff
@@ -1,31 +1,60 @@
...
 const SkillList = ({ className, title, list }) => (
-  <div>
-    <h3 className={`title ${style.skillTitle}`}>{title}</h3>
-    <ul className={style.skillList}>
+  <div className={style.skillsCategory}>
+    <h3 className={`title ${style.skillsCategoryTitle}`}>{title}</h3>
+    <ul className={style.skills}>
       {list.map(tech =>
-        <li key={tech} className={style.skill}>{tech}</li>
+        <li key={tech} className={[`skill skill-${title}`]}>{tech}</li>
```
Comment (ChemicalRocketeer) [inline src/components/UserTile.jsx]: I am by no means an expert in modular css, but here's how I would do this:

To start, I'd make `.skill`, `.skill-Seeking`, and `.skill-Offering` all local. I'd rename `.skill-Seeking` and `.skill-Offering` to just `.Seeking` and `.Offering` Then in the component, I'd do something like: 

```jsx
className={[style.skill, style[title]].join(' ')}
```

Comment (ChemicalRocketeer) [reply src/components/UserTile.jsx]: This all looks great though, if you get the tests passing again I'd be happy to merge this in despite the little nitpicks.

**Comment by kirillrogovoy** [comment]
I think we also need to deal with the filter here if #1 gets merged first.

**Comment by MrTamagotchi** [comment]
Happy that you like it! 👍 
Will definitely get on all of these points ASAP!

File: package.json [hunk 3]
```diff
@@ -46,5 +46,8 @@
     "style-loader": "^0.18.2",
     "webpack": "^3.6.0",
     "webpack-dev-server": "^2.4.1"
+  },
+  "dependencies": {
+    "classnames": "^2.2.5"
```
Comment (ChemicalRocketeer) [inline package.json]: If this is actually used then it's fine but I'd like to avoid unused dependencies

**Comment by MrTamagotchi** [comment]
All issues are now resloved! 🎉

**Comment by MrTamagotchi** [comment]
✨ HOORAY! ✨

**Pull request merged**